.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: 19.10b0
     hooks:
     -   id: black
-        language_version: python3.7
+        language_version: python3
```

Comment:
@TomAugspurger ok with this?

Reply:
Sure. I assume it's just the latest stable?

Reply:
yep sounds right